apps/documentation/src/pages/howToContribute/index.tsx

```diff
@@ -97,16 +124,27 @@ const HowToContribute = (): JSX.Element => (
           you can add links to ressources to help the user understand the rule.
         </p>
       </Block>
-      <Block title='Write the end to end test of your rule in the "test.ts" file'>
+      <Block title='(Optional) Write the end to end test of your rule in the "test.ts" file'>
```

Comment:
Ma proposition pour le block concernant les tests:

First of all, you should test that your rule works as expected on your stack.
Then you can write an end to end test that will check that your rule works on a specially crafted stack. The passing and failing created ressources will be tagged so they can be easily checked. ( You might need to create a new default construct for your specific resource.) 
⚠️Some rules should not be tested:
* If testing the rule requires provisioning a non-serverless resource (for example a RDS or a Lambda with provisioned concurrency) don’t test it. We care about the planet (and our costs). We don’t want to allocate compute or storage units only for our test stack. 
* CloudFormation does not support tags on some resources: SES resources like ConfigurationSet and Identity for example. As you need to tag the created resource to be able to check if the rule works, rules that concern these types of resources cannot be tested.
 In these cases, you can remove the test file.

Reply:
je pense qu'on peut ajouter les bullet points pour que ce soit plus aéré
<img width="1198" alt="Capture d’écran 2023-06-23 à 17 25 40" src="https://github.com/sls-mentor/sls-mentor/assets/91066744/4a8a9efd-680d-4143-8fb4-63cc79c10fa2">